### ISSUE TITLE
Make assets available from Github project

### ISSUE THREAD

**Issue by vigliensoni** [aventures-sonores/aventures-sonores-moon#10]
The project can't access assets from the subproject

For example, when trying to access files (e.g., images) served from the subproject

https://aventuresonores.com/aventures-sonores-moon/artists/

The console returns 
```
15:31:55.727
GEThttps://aventuresonores.com/assets/img/artists/maar.jpg
[HTTP/2 404 Not Found 32ms]

15:31:55.728
GEThttps://aventuresonores.com/assets/img/artists/gb.jpg
[HTTP/2 404 Not Found 32ms]

15:31:55.729
GEThttps://aventuresonores.com/assets/img/artists/vigliensoni.jpg
````

The `config.yml` is properly configured, though

https://github.com/aventures-sonores/aventures-sonores-moon/blob/98eb8bd806f0187dcbf48836bd283a17b3c0f7bc/_config.yml#L11

**Comment by vigliensoni** [comment]
OK, so this was fixed in c08c7bda8b5b0e389871b267e6a22fd09e9f6b9b

@maar34 there is something strange with the configuration of this site, the way local images need to be called from the code is:

```
![Maar]({{ site.url }}/assets/img/artists/maar.jpg)
```

**Comment by vigliensoni** [comment]
@maar34 Y recuerda siempre hacer `git pull` antes de empezar a trabajar para que traigas los últimos cambios y minimizar la posibilidad de conflictos.
